### PR TITLE
client send hdfs user pass

### PR DESCRIPTION
**Pull request by typhoonzero**
Related: https://github.com/sql-machine-learning/sqlflow/issues/1048

### CODE REVIEW

File: sqlflow/client.py [hunk 1]
```diff
@@ -69,6 +69,9 @@ def sql_request(self, sql):
         user_id = os.getenv("SQLFLOW_USER_ID", "")
         hive_location = os.getenv("SQLFLOW_HIVE_LOCATION", "")
         hdfs_namenode_addr = os.getenv("SQLFLOW_HDFS_NAMENODE_ADDR", "")
+        # environment variables JUPYTER_HADOOP_USER, JUPYTER_HADOOP_PASS stores the user's hdfs credentials.
+        hdfs_user = os.getenv("JUPYTER_HADOOP_USER", "")
```
Comment (Yancey0623) [inline sqlflow/client.py:73]: Using the unified prefix `SQLFLOW_` ?

Comment (typhoonzero) [reply sqlflow/client.py:73]: The name `JUPYTER_HADOOP_USER` is used by didi currently.

Comment (typhoonzero) [reply sqlflow/client.py:73]: Should refine this later.

Comment (Yancey0623) [reply sqlflow/client.py:73]: I see..

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**